hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirMkdirOp.java

```diff
@@ -221,8 +221,8 @@ private static INodesInPath unprotectedMkdir(FSDirectory fsd, long inodeId,
     final INodeDirectory dir = new INodeDirectory(inodeId, name, permission,
         timestamp);
 
-    INodesInPath iip =
-        fsd.addLastINode(parent, dir, permission.getPermission(), true);
+    INodesInPath iip = fsd.addLastINode(parent, dir,
+            permission.getPermission(), true, null, true);
```

Comment:
Suggest to keep code format as usual, please also check other changed line. Not blocker issue. Thanks.

Reply:
ok